mobile/pubspec.yaml

```diff
@@ -27,70 +27,67 @@ dependencies:
     sdk: flutter
 
   shared_preferences: ^2.0.15
-  google_maps_flutter: ^2.2.1
+  google_maps_flutter: ^2.2.3
   json_annotation: ^4.7.0
   http: ^0.13.5
-  cached_network_image: ^3.2.2
+  cached_network_image: ^3.2.3
   geolocator: ^9.0.2
-  flutter_local_notifications: ^12.0.3
+  flutter_local_notifications: ^13.0.0
   url_launcher: ^6.1.6
   path_provider: ^2.0.11
   animations: ^2.0.7
-  provider: ^6.0.4
-  charts_flutter: ^0.12.0
+  provider: ^6.0.5
+  charts_flutter: ^0.12.0 # TODO: Use another package. charts_flutter is discontinued. https://pub.dev/packages/charts_flutter
```

Comment:
@BenjaminSsempala @Mnoble-19 take note of this.

Reply:
@NoahNsimbe do you already have a replacement in mind?